libs/libgcrypt/Makefile

```diff
@@ -14,6 +14,7 @@ PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.gnupg.org/gcrypt/libgcrypt
 PKG_HASH:=f9461b4619bb78b273a88d468915750d418e89a3ea3b641bab0563a9af4b04d0
+PKG_LICENSE:=GPL-2.0+
```

Comment:
According to https://gnupg.org/software/libgcrypt/index.html, libgcrypt library was licensed under LGPL-2.1+.  Please also add the `PKG_LICENSE_FILES` definition

Reply:
I am not sure what the right PKG_LICENSE is!
For the lib it is true it is LGPL-2.1+ but for the helper it is GPL-2.0.
But the package builds lib and bin at once.
What is the right Text to add to PKG_LICENSE?
PKG_LICENSE_FILES is only for the bin GPL-2.0

Reply:
Yeah, I also had the same concern.  Looks like we can just specify all the licenses in use in `PKG_LICENSE`.  There is already an example doing this in `include/u-boot.mk` of LEDE repo.